### PR TITLE
LPS-174874 Move Dom4JUtil usage from DynamicQueuryUADExporter into BaseUADExporters

### PR DESCRIPTION
**Pull request by julschong**
resubmitted from #3559

Changes in this pull:
- Fixed testExport and testExportAll Integration tests

### CODE REVIEW

File: modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/base_uad_exporter.ftl [hunk 1]
```diff
@@ -41,25 +45,58 @@ public abstract class Base${entity.name}UADExporter extends DynamicQueryUADExpor
...
 
-		<#list entity.UADEntityColumns as entityColumn>
-			<#if !stringUtil.equals(entityColumn.type, "Blob") || !entityColumn.lazy>
-				sb.append("<column><column-name>${entityColumn.name}</column-name><column-value><![CDATA[");
-				sb.append(${entity.variableName}.get${entityColumn.methodName}());
-				sb.append("]]></column-value></column>");
-			</#if>
-		</#list>
+			Element modelElement = document.addElement("model");
+			Element modelNameElement = modelElement.addElement("model-name");
```
Comment (julschong) [inline modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/base_uad_exporter.ftl:54]: changed this from document.addElement to modelElement.addElement

**Comment by Kvale1733** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:stop

**Comment by Kvale1733** [comment]
@julschong No regressions found.

**Comment by tinatian** [comment]
@julschong As we talked, please resend.

Thanks.
Tina

**Pull request closed without merging**